src/Components/MainComponent/MainComponent.js

```diff
@@ -15,68 +13,88 @@ import {GoogleMapComponent} from '../GoogleMapComponent/GoogleMapComponent'
 import {allCountriesHTTPAddress} from '../../const/const';
 
 export class MainComponent extends Component {
-        constructor(props) {
-            super(props)
-            this.state = {
-                allCountries: [],
-                selectedCountry: {
-                    name: '',
-                    capital: '',
-                    alpha3Code: '',
-                    borders: [],
-                    latlng: [-30, -30]
-                },
-            }
+    constructor(props) {
+        super(props)
+        this.state = {
+            allCountries: [],
+            selectedCountry: {
+                name: '',
+                capital: '',
+                borders: [],
+                latlng: [-30, -30]
```

Comment:
To może zostać wyniesione do jakiejś `const.js`

Reply:
Done.